inc/deployfile.class.php

```diff
@@ -189,6 +188,16 @@ public function displayList(PluginGlpiinventoryDeployPackage $package, $data, $r
                      $pics_path .
                      "uncompress.png' /></a>";
             }
+
+            //download file
+            if (
+                $this->checkPresenceFile($sha512)
+            ) {
```

Comment:
`$file_id` will only be available when this condition is met.
```suggestion
            if (!$fileregistry_error) {
```